### ISSUE TITLE
Example/tutorial notebooks not visible

### ISSUE THREAD

**Issue by dlakaplan** [nanograv/PINT#1137]
The latest builds no longer show the example/tutorial notebooks.  I think this is a result of a change to make them auto-indexed.

**Comment by aarchiba** [comment]
The tutorial notebooks are not part of the documentation generated by autosummary, so nothing should have changed (but it obviously has). Are they visible in local builds (that is, is this an issue specifically with readthedocs, like #1099)?

**Comment by dlakaplan** [comment]
When I do a local `make docs` I see under Tutorials:
* Data Files
* Examples
  * Basic installation
  * Validation Example for PINT paper

This is the same as I see in the "latest" on readthedocs.  If I go to "stable" I see the expected suite of tutorials (those listed in `docs/tutorials.rst`).

**Comment by aarchiba** [comment]
Ah. Well at least it's replicable locally, that should make debugging easier! In fact there are probably messages in the `make docs` output (buried among the others) listing the file names and saying what's wrong with them.

**Comment by dlakaplan** [comment]
So during the local build I see (and have ignored):
```
/Users/kaplan/pythonpackages/PINT_dlakaplan/docs/examples/build_model_from_scratch.py:28: WARNING: Unexpected indentation.
/Users/kaplan/pythonpackages/PINT_dlakaplan/docs/examples/build_model_from_scratch.py:29: WARNING: Block quote ends without a blank line; unexpected unindent.
/Users/kaplan/pythonpackages/PINT_dlakaplan/docs/examples/build_model_from_scratch.py:31: WARNING: Unexpected indentation.
/Users/kaplan/pythonpackages/PINT_dlakaplan/docs/examples/build_model_from_scratch.py:33: WARNING: Block quote ends without a blank line; unexpected unindent.
...
```

and then:
```
...
/Users/kaplan/pythonpackages/PINT_dlakaplan/docs/tutorials.rst:35: WARNING: toctree contains reference to document 'examples/build_model_from_scratch' that doesn't have a title: no link will be generated
...
```

**Comment by dlakaplan** [comment]
When I do this locally, I can get it to work if I do a `make notebooks` before `make docs`.  So somehow the notebooks are not being generated from the python.  With the `make notebooks` first, I also avoid the warnings like:
```
...
/Users/kaplan/pythonpackages/PINT_dlakaplan/docs/tutorials.rst:35: WARNING: toctree contains reference to document 'examples/build_model_from_scratch' that doesn't have a title: no link will be generated
...
```
@aarchiba : does this make any sense?

**Comment by scottransom** [comment]
I tried this today locally multiple times and it seems to work for me no matter what order I do things. And I was pretty careful to clean things up between tries.

**Comment by aarchiba** [comment]
Closed by #1154
